flow/aggregates.py

```diff
@@ -430,57 +461,22 @@ class _MakeAggregates:
         The ``select`` attribute associated with the :py:class:`aggregator`
     :type select:
         callable or NoneType
-    :param project:
-        A signac project used to fetch jobs for creating aggregates.
-    :type project:
-        :py:class:`flow.FlowProject` or :py:class:`signac.contrib.project.Project`
     """
-    def __init__(self, aggregator_function, sort_by, reverse_order, select, project):
+    def __init__(self, aggregator_function, sort_by, reverse_order, select):
         self._aggregator_function = aggregator_function
         self._sort_by = sort_by
         self._reverse_order = bool(reverse_order)
         self._select = select
-        self._project = project
-
-    def __call__(self):
-        jobs = list(self._project)
 
+    def __call__(self, project):
+        jobs = list(project)
```

Comment:
This list call is unnecessary. If we require that the aggregator function be passed a list, then we should only call list if `self._sort_by is None`.

````python
if self._sort_by is None:
    jobs = list(jobs)
else:
    jobs = sorted(...)